src/test/java/com/acme/conferencesystem/ArchitectureTests.java

```diff
@@ -22,7 +22,9 @@ void verifyModularStructure() {
     void writeDocumentation() {
         new Documenter(modules).writeDocumentation();
 
-        //new Documenter(modules).writeModulesAsPlantUml().writeIndividualModulesAsPlantUml().writeModuleCanvases();
+        new Documenter(modules)
+                .writeDocumentation()
+                .writeIndividualModulesAsPlantUml();
```

Comment:
comment or delete this xD

new Documenter(modules).writeDocumentation(); this does everything and the commented code is what it does

Reply:
Have a look of the implementation of writeDocumentation